docs/run/error-mitigation-explanation.ipynb

```diff
@@ -0,0 +1,274 @@
+    "This page provides high-level explanations of the error suppression and error mitigation techniques available through Qiskit Runtime.\n",
+    "\n",
+    "The following cell imports the Estimator primitive and creates a backend that will be used for initializing the Estimator in later code cells."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "39ef65e5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_ibm_runtime import EstimatorV2 as Estimator\n",
+    "from qiskit_ibm_runtime import QiskitRuntimeService\n",
+    "\n",
+    "service = QiskitRuntimeService()\n",
+    "backend = service.least_busy()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ad548ef5-f1c1-4def-b551-b3ece2032065",
+   "metadata": {
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "source": [
+    "## Dynamical decoupling\n",
+    "\n",
+    "Quantum circuits are executed on IBM hardware as sequences of microwave pulses that need to be scheduled and run at precise time intervals.\n",
+    "Unfortunately, unwanted interactions between qubits can lead to coherent errors on idling qubits. Dynamical decoupling works by inserting pulse sequences on idling qubits to approximately cancel out the effect of these errors. Each inserted pulse sequence amounts to an identity operation, but the physical presence of the pulses has the effect of suppressing errors.\n",
+    "There are many possible cohoice of pulse sequences, and which sequence is better for each particular case remains an [active area of research](https://journals.aps.org/prapplied/abstract/10.1103/PhysRevApplied.20.064027).\n",
+    "\n",
+    "The diagram below depicts dynamical decoupling with an XX pulse sequence. The abstract circuit on the left is mapped onto a microwave pulse schedule on the top right. The bottom right depicts the same schedule, but with a sequence of two X pulses inserted during an idle period of the first qubit.\n",
+    "\n",
+    "![Depiction of dynamical decoupling](/images/run/error-mitigation-explanation/dd.jpg)\n",
+    "\n",
+    "The Qiskit Runtime options for dynamical decoupling are described [here](https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.DynamicalDecouplingOptions). Notably, the `sequence_type` option can be used to pick from several different pulse sequences.\n",
+    "\n",
+    "The following code cell shows how to enable dynamical decoupling for the estimator and choose a dynamical decoupling sequence."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "cc057e3f-f592-434e-8bb2-96f901564318",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true,
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "estimator = Estimator(backend=backend)\n",
+    "estimator.options.dynamical_decoupling.enable = True\n",
+    "estimator.options.dynamical_decoupling.sequence_type = \"XpXm\""
```

Comment:
is setting the sequence type optional or required? I think it would be good to mention either way

Reply:
Also I notice in the other sections you explicitly say that to enable the technique you must set enable = true but you dont say that here. I think you should either say that point for all of them or none of them, but just make sure it's consistent